google-cloud-error_reporting/test/google/cloud/error_reporting/error_event_test.rb

```diff
@@ -86,7 +86,7 @@
       exception.set_backtrace([])
       error_event =
         Google::Cloud::ErrorReporting::ErrorEvent.from_exception exception
-      error_event.message.must_equal exception_message
+      error_event.message.must_match exception_message
```

Comment:
Why was this expectation changed? Was it failing?

Reply:
Yes.
When casting string, Error message changes `A serious error from application` to `: A serious error from application (StandardError))\n\t`